install_cryosparc_master.sh

```diff
@@ -43,5 +43,4 @@ export master_hostname=$(echo -e "$(hostname -f)" | tr -d '[:space:]')
 # submitting a job because the system Python has another version and its
 # packages cannot be found. The temporary workaround is to link required
 # Python packages (slurm_foz) in the cryoSPARC conda environment
-ln -s /usr/lib/python3.6/site-packages/slurm_foz \
-    ${SPARCDIR}/cryosparc_master/deps/anaconda/envs/cryosparc_master_env/lib/python3.8/site-packages/
+# ln -s /usr/lib/python3.6/site-packages/slurm_foz \${SPARCDIR}/cryosparc_master/deps/anaconda/envs/cryosparc_master_env/lib/python3.8/site-packages/
```

Comment:
I need to check if this linking is still required on our cluster. On Hortense it will indeed not be necessary.

Reply:
Sure. I have commented it out.

Reply:
I checked and on our cluster this hack is still required to make `sam-balance` work when submitting a job from cryoSPARC. Without the hack, the job is still queued but the check for credits is skipped.

Perhaps we can have replace this line with:
```
if [ -d /usr/lib/python3.6/site-packages/slurm_foz ]; then
    ln -s /usr/lib/python3.6/site-packages/slurm_foz \
    ${SPARCDIR}/cryosparc_master/deps/anaconda/envs/cryosparc_master_env/lib/python3.8/site-packages/
fi
```
so it works on both our cluster as on others.

Reply:
fixed in fbf21d1